shared/more/dumb-sheet.render.desktop.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react'
+import {Box, Text} from '../common-adapters'
+import {globalStyles} from '../styles/style-guide'
+import type {DumbMap} from './dumb'
+
+import CommonMap from '../common-adapters/dumb'
+import LoginMap from '../login/dumb'
+import SignupMap from '../login/signup/dumb'
+import TrackerMap from '../tracker/dumb'
+import PinentryMap from '../pinentry/dumb'
+
+class Render extends Component {
+  render () {
+    const componentMap: DumbMap = {
+      ...CommonMap,
+      ...LoginMap,
+      ...SignupMap,
+      ...TrackerMap,
+      ...PinentryMap
+    }
+
+    return (
```

Comment:
we could make this fancier later but for now just blast it all into one thing. easier to screenshot it this way